.github/workflows/frontend.yml

```diff
@@ -30,6 +30,18 @@ jobs:
           path: |
             **/node_modules
           key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: build cache
+        if: |
+          steps.node-modules-cache.outputs.cache-hit == 'true' &&
+          github.ref != 'refs/heads/main'
+        id: build-cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            **/dist
+            **/tsconfig.tsbuildinfo
+            !**/node_modules
+          key: ${{ runner.os }}-build-${{ hashFiles('**/yarn.lock') }}
```

Comment:
use a more specific key, i would change the `modules` one above too

Reply:
Updated on https://github.com/lyft/clutch/pull/1288/commits/e44bbc7c550eb8dd9224b129c3550d342ec220b1